data/calculated.py

```diff
@@ -128,26 +128,10 @@ def __init__(self, parent, expression, dims, attrs={}, db_url=""):
             pass
 
     def __getitem__(self, key):
-        """This is where the magic happens.
-        The dimensions of the key are determined by looking at the dimensions
-        of all the underlying variables used in the calculation, this is then
-        passed along to the parser where the calculation is performed.
-        """
-        key_dims = ()
```

Comment:
Delete all this stuff since we explicitly define the dims in the dataset config a la #630